migrations/0001_schema.sql

```diff
@@ -95,4 +95,11 @@ CREATE TABLE embedding(
 
     PRIMARY KEY (id),
     FOREIGN KEY(chunk_id) REFERENCES chunk (id)
-);
\ No newline at end of file
+);
+
+-- Default collection
+INSERT INTO collection (name, text_embedding_model, text_distance_metric) VALUES ('main', 'openai:text-embedding-ada-002', 'cosine');
```

Comment:
Thought -- put this in a separate script and make it a flag as to whether we apply it (eg., call this testdata injection or something)? That would make it easier to run with/without it.

Also: this is why I take the sha256 of the schema file -- we're going to change it, and when we do that let's us know "the DB has already applied a different version of `0001_schema.sql`".